### ISSUE TITLE
Browsing sorted table shows duplicate rows - misses others while paging

### ISSUE THREAD

**Issue by NKarasek** [phpmyadmin/phpmyadmin#16143]
### Describe the bug
Table with 220 entries.  A column has six distinct values.  Sorting by that column displaying 50 rows/table lists the first 50 rows correctly.  Moving to page 2 repeats the last 5 rows from the previous page NOT displaying 5 rows that should be displayed - i.e., the correct count of groups within the sort is retained - but the content is not correct.  Changing to SHOW ALL corrects the display.  All the rows displayed after a break in the sorted column ('c4' => 'c5' - on page 1) are duplicated on page 2 - although do not display in the same order or place within the list.

### To Reproduce

1. Create a table with at least 70 rows.
2. Create a `columnX` with values in at least three groups with one group encompassing at least 16 rows. 
3. Browse the table.
4. Sort ascending by `columnX`
5. Scroll to the bottom of page 1 and save/record the last 10 rows.
6. Advance to page two.
7. Within the first 10 rows will be duplicated row from page 1.

### Expected behavior
A correct display of rows while paging through a sorted browse request (with no filters)

### Screenshots

**Page 1**
![image](https://user-images.githubusercontent.com/4554076/82904350-5914bf80-9f30-11ea-8a93-62e57e13a522.png)

**Page 2**
![image](https://user-images.githubusercontent.com/4554076/82904400-6f228000-9f30-11ea-80cc-cb38cb55f5dd.png)

### Server configuration
Apache/2.4.41 (Win64) OpenSSL/1.1.1c PHP/7.3.10
Database client version: libmysql - mysqlnd 5.0.12-dev - 20150407
PHP extension: mysqli Documentation curl Documentation mbstring Documentation

- PHP version: 7.3.10
- phpMyAdmin version: v-5.0.2 (also 5.1.0 and 5.0.1)

### Client configuration

- Browser:  Chrome  Version 81.0.4044.138 (Official Build) (64-bit)
- Operating system: Windows 10 - 1903 all updates

### Additional context
To verify the table was still OK, I;
1.  exported the table
2.  dropped the table
3.  imported the exported .SQL file.
4.  retested
5.  SAME RESULT -  error in page two displaying 50 rows/page (or page 3 if displaying 25 rows/page)

[properties.zip](https://github.com/phpmyadmin/phpmyadmin/files/4682463/properties.zip)

**Comment by williamdes** [comment]
Thank you for the detailed example @NKarasek !

Maybe related to #14883 or #15389

**Comment by shucon** [comment]
Starting to work on this.

**Comment by shucon** [comment]
I've looked into the code and I think nothing's wrong from PMA's side. It might be a bug in MySQL because the rows are being repeated even while using MySQL terminal.
Just to be sure, is the query supposed to be like this?
`SELECT * FROM properties ORDER BY prConnector ASC LIMIT 0, 50` - for page 1
`SELECT * FROM properties ORDER BY prConnector ASC LIMIT 50, 50` - for page 2
and so on.

**Comment by williamdes** [comment]
> I've looked into the code and I think nothing's wrong from PMA's side. It might be a bug in MySQL because the rows are being repeated even while using MySQL terminal.
> Just to be sure, is the query supposed to be like this?
> `SELECT * FROM properties ORDER BY prConnector ASC LIMIT 0, 50` - for page 1
> `SELECT * FROM properties ORDER BY prConnector ASC LIMIT 50, 50` - for page 2
> and so on.
> @ibennetch @mauriciofauth @williamdes

I would say yes, it seems fine. Maybe some obscure caching option somewhere.

**Comment by NKarasek** [comment]
I think I have deduced the problem.  It appears that caching is not involved as I disabled all caching options, retested, and saw the same "error".  Looking at the two queries shown above, the ONLY column being sorted is prConnector, which allows rows that are in each group to appear in whatever order mysql finds them.  If the second query using LIMIT 50, 50, does a **full retrieval** and then simply skips the first 50, there is no guarantee that the second group of 50 is actually an identical copy of the result from the first query.  As long as the rows satisfy the requirement of being sorted by prConnector, mysql has done its job correctly.  Adding the SECOND sort key of prLegalID to both queries forces the collections to be in order by both columns - which then displays page 1, page 2  . . .  page n  as expected.  I think I can say that this 'BUG' is actually a User Error (mine)  and should probably be closed.  :-(
